[serverless] recommend extension over forwarder (#631)
DataDog/datadog-serverless-functions: aws/logs_monitoring/README.md

  - /serverless/libraries_integrations/forwarder/
---

The Datadog Forwarder is an AWS Lambda function that ships logs from AWS to Datadog, specifically:

- Forward CloudWatch, ELB, S3, CloudTrail, VPC, SNS, and CloudFront logs to Datadog
- Forward S3 events to Datadog
- Forward Kinesis data stream events to Datadog (only CloudWatch logs are supported)

> - Forward metrics, traces and logs from AWS Lambda logs to Datadog (we recommend using the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) instead)

Reviewer: ```suggestion
- [Legacy] Forward metrics, traces and logs from AWS Lambda functions to Datadog (we now recommend using the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) to monitor your Lambda functions)
```

> - Forward metrics, traces and logs from AWS Lambda logs to Datadog (we recommend using the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) instead)

Reviewer 2: ```suggestion
- Forward metrics, traces, and logs from AWS Lambda functions to Datadog. **Note**: Datadog recommends that you use the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) to monitor your Lambda functions.
```

applying our styleguide and also a serial comma to ryan's suggestion.

> - [Legacy] Forward metrics, traces and logs from AWS Lambda functions to Datadog (we now recommend using the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) to monitor your Lambda functions)

Reviewer 2: ```suggestion
- Forward metrics, traces, and logs from AWS Lambda functions to Datadog. **Note**: Datadog recommends that you use the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) to monitor your Lambda functions.
```

For additional information on sending AWS services logs with the Datadog Forwarder, read the [Send AWS Services Logs with the Datadog Lambda Function](https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/) guide.

> For serverless customers using the Forwarder to forward metrics, traces and logs from AWS Lambda logs to Datadog, we recommend [migrating to the Datadog Lambda Extension](https://docs.datadoghq.com/serverless/guide/extension_motivation/) to collect telemetry directly from the Lambda execution environments. The Forwarder is still available for use, but will only receive security updates.

Reviewer 2: ```suggestion
For serverless customers using the Forwarder to forward metrics, traces, and logs from AWS Lambda logs to Datadog, it is recommended that you [migrate to the Datadog Lambda Extension](https://docs.datadoghq.com/serverless/guide/extension_motivation/) to collect telemetry directly from the Lambda execution environments. The Forwarder is still available for use, but will only receive security updates.
```
